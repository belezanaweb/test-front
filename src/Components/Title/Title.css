.Title {
  color: #999;
  font-family: 'Helvetica Neue';
  font-size: 14px;
  font-weight: 700;
  line-height: 17px;
  margin-left: 11px;
  margin-bottom: 5px;
}
