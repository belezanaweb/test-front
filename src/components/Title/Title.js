import styled from 'styled-components'
import theme from 'theme'

const Title = styled.h1`
  padding: 0 0.8462rem;
  text-transform: uppercase;
  color: #999;
  font-weight: ${theme.fontWeightBold};
  font-size: 1.077rem;
  margin-bottom: 0.3846rem;
`

export default Title
