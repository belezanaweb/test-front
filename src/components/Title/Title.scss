@import '../../assets/sass/colors';

.title {
  width: 100%;
  height: auto;
  display: block;
  padding: 10px 10px 5px 10px;
  &__text {
    font-size: 14px;
    text-transform: uppercase;
    font-weight: 700;
    color: $darkGrayColor;
  }
}
