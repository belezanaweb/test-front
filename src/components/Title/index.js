import styled from 'styled-components'
import px2vw from '../../utils/px2vw'

const Title = styled.p`
  text-transform: uppercase;
  color: #999;
  font-weight: bold;

  margin-left: ${px2vw(20)};
  margin-bottom: ${px2vw(10)};
`

export default Title
