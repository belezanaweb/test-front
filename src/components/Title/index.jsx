import styled from "@emotion/styled";


export const Title = styled.span`
  font-size: 0.875rem;
  font-weight: 700;
  color: ${ ({ theme }) =>  theme.colors.greyDark };
  text-transform: uppercase;
  margin-left: 5px;
  margin-bottom: 5px;
  line-height: 1rem;
`