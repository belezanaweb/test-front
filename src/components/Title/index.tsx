import React from 'react';

import { Container } from './styles';

const Title: React.FC = ({ children }) => {
  return <Container>{children}</Container>;
};

export default Title;
