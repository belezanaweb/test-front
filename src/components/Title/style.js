import styled from 'styled-components'

export const Container = styled.h4`
  padding: 0 10px;
  font-size: 14px;
  font-weight: 800;
  color: #999;
  margin-bottom: 5px;
  text-transform: uppercase;
  @media screen and (min-width: 1024px) {
    font-size: 1rem;
  }
`
