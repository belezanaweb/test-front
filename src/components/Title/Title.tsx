import React, { FC } from 'react'

import * as S from './Title.styled'

export const Title: FC = (props) => <S.Title>{props.children}</S.Title>
