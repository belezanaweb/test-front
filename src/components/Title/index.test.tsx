import { render } from '@testing-library/react';
import React from 'react';
import Title from '.';

describe('<Title />', () => {
  it('should be able to render correctly', () => {
    // Arrange
    const { baseElement } = render(<Title>Text</Title>);

    // Assert
    expect(baseElement).toMatchSnapshot();
  });
});
