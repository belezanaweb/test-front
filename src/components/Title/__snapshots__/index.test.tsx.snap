// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should be able to render correctly 1`] = `
<body>
  <div>
    <h2
      class="sc-bdnylx hpwNdB"
    >
      Text
    </h2>
  </div>
</body>
`;
